release/release.go

```diff
@@ -1196,6 +1196,7 @@ func linuxRelease(v version.Version) {
 							"--username", os.Getenv("BARQUE_USERNAME"),
 							"--api_key", os.Getenv("BARQUE_API_KEY"),
 						}
+						log.Printf("curatorArgs %v\n", curatorArgs)
```

Comment:
printing the args directly is problematic, because we will print the barque API key -- we probably need to do something like replace the API key with `XXXXX` if we're going to log this